### PR TITLE
docs: a few fixes, add no empty blocks

### PR DESCRIPTION
**Pull request by zramsay**
Some docs fixes, relates to #350

### CODE REVIEW

File: docs/getting-started/getting-started.rst [hunk 1]
```diff
@@ -24,13 +24,6 @@ these instructions.
 
         ethermint --datadir ~/.ethermint init
 
-* Note: You can optionally copy a keystore to the Ethereum folder that you used in the steps above i.e `~/.ethermint` e.g
```
Comment (melekes) [inline docs/getting-started/getting-started.rst:27]: mind me asking why are we deleting this?

File: docs/getting-started/getting-started.rst [hunk 2]
```diff
@@ -46,6 +39,19 @@ go-ethereum functionality.
 
 The **password** for the default account is *1234*, which we'll need below.
 
+For development purposes, you can enable tendermint to only produce blocks when there are transactions. Add to the
```
Comment (melekes) [inline docs/getting-started/getting-started.rst:42]: this does not work currently, right? https://github.com/tendermint/ethermint/issues/308 so isn't it too early to document this?

Comment (zramsay) [reply docs/getting-started/getting-started.rst:42]: #350 was merged

File: docs/getting-started/getting-started.rst (hunk 1 above)
Comment (zramsay) [reply docs/getting-started/getting-started.rst:27]: because the copying happens automatically (see #245)

Comment (zramsay) [reply docs/getting-started/getting-started.rst]: https://github.com/tendermint/ethermint/pull/245

**Pull request merged**